src/CommitmentsV2/SFA.DAS.CommitmentsV2/Domain/Interfaces/ICurrentDateTime.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace SFA.DAS.CommitmentsV2.Domain.Interfaces
+{
+    public interface ICurrentDateTime
+    {
+        DateTime Now { get; }
```

Comment:
Do you think UtcNow might be a better name? Currently looking at this devs are going to think it is analogous to DateTime.Now when in fact it is UtcNow (i.e. without time zone adjustment). Probably the same might apply to the  CurrentDateTime config property.